docs/concepts/what-is-a-temporal-cluster.md

```diff
@@ -66,12 +66,13 @@ The History Service scales horizontally via individual shards, configured during
 The number of shards remains static for the life of the Cluster (so you should plan to scale and over-provision).
 
 Each shard maintains data (routing Ids, mutable state) and queues.
-There are three types of queues that a History shard maintains:
+There are four types of queues that a History shard maintains:
```

Comment:
```suggestion
A History shard maintains four types of queues:
```
"There are" and "There is" can almost always be rewritten to produce a stronger sentence. (I had a high skilled technical editor pound that into my neurons years ago.)